packages/core/src/layout/Page/Page.tsx

```diff
@@ -32,14 +30,17 @@ const useStyles = makeStyles(() => ({
 }));
 
 type Props = {
-  theme?: PageTheme;
+  pageTheme?: PageTheme;
```

Comment:
Tbh I don't think we wanna change the name of this prop. Prolly makes sense to keep as just `theme`, since we're already in the `Page` component?